NMWeb/src/app/about-page/about-info/about-info.component.html

```diff
@@ -0,0 +1,92 @@
+<mat-card>
+  <div class="about__basedOn">
+    <mat-card-title>This app connects people based on</mat-card-title>
+    <mat-card-content>
+      <div
+        class="about__basedOn--tiles"
+        fxLayout="row"
+        fxLayout.lt-sm="column"
+        fxLayoutAlign.lt-sm="center center"
+        fxLayoutAlign="center stretch"
+        fxLayoutGap="20px"
+        fxLayoutGap.lt-sm="0px"
+      >
+        <div
+          class="about__basedOn--tile"
+          fxLayout="column"
+          fxLayoutAlign="space-between center"
+        >
+          <div>
+            <mat-icon
+              aria-hidden="false"
+              aria-label="question-answer icon"
+              class="about__basedOn--icon"
+              >question_answer</mat-icon
+            >
+            <h4>Actionable interests</h4>
+            <p>
+              Actionable interests like pair programming, do a hackathon,
+              freelance, co-founder, business partner etc.
+            </p>
+          </div>
+          <a mat-raised-button routerLink="/profile">
+            <span fxLayout fxLayoutAlign="center center" fxLayoutGap="5px">
+              <mat-icon aria-hidden="false" aria-label="event icon"
+                >person_pin</mat-icon
+              >
+              <span>PROFILE</span>
+            </span>
+          </a>
+        </div>
+        <div
+          class="about__basedOn--tile"
+          fxLayout="column"
+          fxLayoutAlign="space-between center"
+        >
+          <div>
+            <mat-icon
+              aria-hidden="false"
+              aria-label="location icon"
+              class="about__basedOn--icon"
+              >location_on</mat-icon
+            >
+            <h4>Geo-locations</h4>
+            <p>See users on the map.</p>
+          </div>
+          <a mat-raised-button routerLink="/map">
+            <span fxLayout fxLayoutAlign="center center" fxLayoutGap="5px">
+              <mat-icon aria-hidden="false" aria-label="event icon"
+                >map</mat-icon
```

Comment:
I am not sure about this style of closing elements, it somehow looks confusing to me - at first glance it seemed to me like a typo :P Can you please explain the reasoning behind please if any?

Reply:
I used Prettier to format the code. It looks a bit weird I agree. Can you suggest an alternative?

Reply:
> I used Prettier to format the code. It looks a bit weird I agree. Can you suggest an alternative?

Btw, wrapping margin should be at about ~120-140 chars. So maybe this could be on single line. But let's timebox this to no more than 1 person-minute more :). HTML can be tricky to indent/format sometimes ;).

Reply:
Ok. Do you want me to do another pull request with changed formatting?

Reply:
What IDE do you use? I work using jetbrains stack (webstorm) and it has built in formatter.

Reply:
I use Visual Studio Code. I have Prettier and Beautify extensions and now I can see that Beautify formats HTML in the correct way.

Reply:
Yeah so we can try next time with beautify. As this is minor, it is safe to merge :) Thank you for your contribution!

Reply:
Great :) Thanks for taking your time to do the review :)